### PR TITLE
fix(components): [color-picker] optimize the flickering issue

### PR DESCRIPTION
**Pull request by momei-LJM**
Continuously clicking on the input box panel will cause it to blink.
I think he should be allowed to click again to close it(maybe).
before:
screenshots

after: 
screenshots

### CODE REVIEW

File: packages/components/color-picker/src/color-picker.vue [hunk 1]
```diff
@@ -244,8 +247,8 @@ function setShowPicker(value: boolean) {
   showPicker.value = value
 }
 
+// const debounceSetShowPicker = debounce(setShowPicker, 100, { leading: true })
```
Comment (btea) [inline packages/components/color-picker/src/color-picker.vue:250]: Why write the same code comment here?

Comment (momei-LJM) [reply packages/components/color-picker/src/color-picker.vue]: sorry,I've removed it

**Review by tolking — APPROVED** [review]

**Comment by btea** [comment]
Can you provide a detailed explanation? Based on the example you provided, I don't seem to understand the difference between before and after the modification.

**Comment by momei-LJM** [comment]
> Can you provide a detailed explanation? Based on the example you provided, I don't seem to understand the difference between before and after the modification.

When I click the second time, I can't fold the drop-down selector; But the flicker turned on again

**Review by btea — APPROVED** [review]

**Pull request merged**